gunicorn_starter.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-poetry run gunicorn --chdir todo_app 'todo_app.app:create_app()' -b 0.0.0.0:5000 --access-logfile /var/log/gunicorn-access.log --error-logfile /var/log/gunicorn-error.log
+poetry run gunicorn --chdir todo_app 'todo_app.app:create_app()' -b 0.0.0.0:$PORT --access-logfile /var/log/gunicorn-access.log --error-logfile /var/log/gunicorn-error.log
```

Comment:
You can keep a fallback value of 5000 with `${PORT:-5000}`. That way developers can still run the production image locally without worrying about PORT. Otherwise it should be documented in the README or .env.template